### PR TITLE
It's the tiny things that make a difference

### PR DESCRIPTION
**Pull request by ramico**

### CODE REVIEW

File: includes/class-wc-siftscience-options.php [hunk 1]
```diff
@@ -21,25 +21,24 @@ class WC_SiftScience_Options {
...
 		public const STATS_API   = 'https://sift.fermiac.staat.us';
 
-		public const THRESHOLD_GOOD    = self::SCHEMA . 'threshold_good';
+		public const IS_API_SETUP      = self::SCHEMA . 'is_api_setup';
 		public const THRESHOLD_BAD     = self::SCHEMA . 'threshold_bad';
-		public const AUTO_SEND_ENABLED = self::SCHEMA . 'auto_send_enabled';
-		public const MIN_ORDER_VALUE   = self::SCHEMA . 'min_order_value';
 		public const LOG_LEVEL_KEY     = self::SCHEMA . 'log_level';
-		public const IS_API_SETUP      = self::SCHEMA . 'is_api_setup';
+		public const THRESHOLD_GOOD    = self::SCHEMA . 'threshold_good';
```
Comment (nabsul) [inline includes/class-wc-siftscience-options.php:35]: What is the reason for re-ordering these lines? For example, it now seems strange to me that THRESHOLD_BAD     and THRESHOLD_GOOD   are separated.

Comment (ramico) [reply includes/class-wc-siftscience-options.php:35]: longer line comes l8r

Comment (nabsul) [reply includes/class-wc-siftscience-options.php:35]: I think that's ok sometimes, but in this case better to keep things logically grouped/nearby.

**Comment by ramico** [comment]
There only dictionary but ok

**Pull request merged**